src/EPR.SubsidiaryBulkUpload.Function/CompaniesHouseDownloadFunction.cs

```diff
@@ -1,20 +1,20 @@
-﻿using EPR.SubsidiaryBulkUpload.Application.Services.CompaniesHouseDownload;
+﻿using EPR.SubsidiaryBulkUpload.Application.Services.CompaniesHouseDownload.Interfaces;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
 
 namespace EPR.SubsidiaryBulkUpload.Function;
 
 public class CompaniesHouseDownloadFunction(ICompaniesHouseDownloadService companiesHouseDownloadService, ILogger<CompaniesHouseDownloadFunction> logger)
 {
-    private readonly ICompaniesHouseDownloadService companiesHouseDownloadService = companiesHouseDownloadService;
+    private readonly ICompaniesHouseDownloadService _companiesHouseDownloadService = companiesHouseDownloadService;
```

Comment:
I can see most of the refactoring is using underscore as prefix of each private variable.  This practice is adopted by .NET CoreFX team for their system level coding practices. 

We need to bring this coding convention formally onboard. If all agreed then it should be part of developer pack as well as in Confluence pages. This is important to highlight frequently as developers are different conventions. 

Once the formal Coding Convention is upload in Confluence devs can then follow all the rules and convention in there. For example dev might still be using "This notation" rather than Underscore notation. As some claimed using underscore clutters names. 

Similarly there are other rules like using camelCase for internal and private fields and use readonly where possible. Also starting static fields with s_ and thread static fields with t_. When used on static fields, readonly should come after static (i.e. static readonly not readonly static) etc.

Reply:
Defra coding standards are already available in github. Making people follow them is another matter - but if you see people not following standard naming then you can remind them when reviewing PRs.